Source/Core/DolphinQt/SkylanderPortal/SkylanderModifyDialog.cpp

```diff
@@ -0,0 +1,317 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "SkylanderModifyDialog.h"
+
+#include <Core/IOS/USB/Emulated/Skylanders/Skylander.h>
+#include <Core/System.h>
+
+#include <DolphinQt/QtUtils/SetWindowDecorations.h>
+
+SkylanderModifyDialog::SkylanderModifyDialog(QWidget* parent, u8 slot) : QDialog(parent)
+{
+  this->slot = slot;
+
+  layout = new QVBoxLayout;
+
+  IOS::HLE::USB::Skylander* skylander =
+      Core::System::GetInstance().GetSkylanderPortal().GetSkylander(slot);
+
+  figure = skylander->figure.get();
+  figureData = figure->GetData();
+
+  auto* hbox_name = new QHBoxLayout;
+  QString name = QString();
+
+  if ((figureData.skylanderData.nickname[0] != 0x00 &&
+       figureData.normalizedType == static_cast<u8>(IOS::HLE::USB::Type::Skylander)))
+  {
+    name = tr("\"%1\"").arg(
+        QString::fromUtf16(reinterpret_cast<char16_t*>(figureData.skylanderData.nickname.data())));
```

Comment:
You should be able to remove the `tr` call as nothing is going to be translated anyways.